packages/gatsby-cli/src/create-cli.js

```diff
@@ -178,7 +178,12 @@ function buildLocalCommands(cli, isLocalSite) {
         .option(`o`, {
           alias: `open`,
           type: `boolean`,
-          describe: `Open the site in your browser for you.`,
+          describe: `Open the site in your (default) browser for you.`,
+        })
+        .option(`prefix-paths`, {
+          type: `boolean`,
+          default: false,
+          describe: `Serve site with link paths prefixed (set pathPrefix in your gatsby-config.js.`,
```

Comment:
This should clarify this will only work if you built the site previously with the flag set